app/src/main/java/com/example/androidadvancedproject/data/LodgeItemDTO.java

```diff
@@ -0,0 +1,45 @@
+package com.example.androidadvancedproject.data;
+
+import java.util.Date;
+
+public class LodgeItemDTO {
+     private final int lodge_id;
+     private final String lodge_name;
+     private final String lodge_owner;
+     private final Date availability;
+     private final String lodge_number;
```

Comment:
use camel case as a naming convention. try to avoid using underscores, unless naming constants